### ISSUE TITLE
About the division of OfficeHome dataset

### ISSUE THREAD

**Issue by TomSheng21** [MediaBrain-SJTU/FACT#2]
Thanks for your work. There is no official division of the OfficeHome dataset. How do you divide OfficeHome into a training set and a validation set. Is the result in your paper the result of the test set corresponding to the highest accuracy of the validation set? Thanks

**Comment by TomSheng21** [comment]
And did you use ColorJitter in DeepAll? If I use ColorJitter  in my DeepAll experiment, PACS Sketch should reach 75 or more

**Comment by PPJ001** [comment]
Hello, can I add a contact way to communicate with you? I'd like to ask you some questions about the experiment
